packages/wonder-blocks-dropdown/src/components/option-item.tsx

```diff
@@ -124,93 +170,143 @@ export default class OptionItem extends React.Component<OptionProps> {
 
+        const defaultStyle = [
+            styles.item,
+            // pass optional styles from react-window (if applies)
+            style,
+        ];
+
         return (
-            <ClickableBehavior
+            <DetailCell
                 disabled={disabled}
-                onClick={this.handleClick}
+                horizontalRule={horizontalRule}
+                rootStyle={defaultStyle}
+                style={styles.itemContainer}
+                aria-selected={selected ? "true" : "false"}
                 role={role}
-                tabIndex={0}
-            >
-                {(state, childrenProps) => {
-                    const {pressed, hovered, focused} = state;
-
-                    const defaultStyle = [
-                        styles.itemContainer,
-                        pressed
-                            ? styles.active
-                            : (hovered || focused) && styles.focus,
-                        disabled && styles.disabled,
-                        // pass optional styles from react-window (if applies)
-                        style,
-                    ];
-
-                    return (
-                        <View
-                            {...sharedProps}
-                            testId={testId}
-                            style={defaultStyle}
-                            aria-selected={selected ? "true" : "false"}
-                            role={role}
-                            {...childrenProps}
-                        >
+                testId={testId}
+                leftAccessory={
+                    <>
+                        {leftAccessory ? (
+                            <View style={{flexDirection: "row"}}>
+                                <CheckComponent
+                                    disabled={disabled}
+                                    selected={selected}
+                                />
+                                <Strut size={Spacing.xSmall_8} />
+                                {leftAccessory}
+                            </View>
+                        ) : (
                             <CheckComponent
                                 disabled={disabled}
                                 selected={selected}
-                                pressed={pressed}
-                                hovered={hovered}
-                                focused={focused}
                             />
-                            <LabelMedium style={styles.label}>
-                                {label}
-                            </LabelMedium>
-                        </View>
-                    );
-                }}
-            </ClickableBehavior>
+                        )}
+                    </>
+                }
+                rightAccessory={rightAccessory}
+                subtitle1={
+                    <LabelSmall className="subtitle">{subtitle1}</LabelSmall>
+                }
```

Comment:
**note:** Adding this custom "LabelSmall.subtitle" wrapper to be able to style the hover state correctly.